channels/test_utils.py

```diff
@@ -0,0 +1,18 @@
+"""utils for dealing with channel tests"""
+from functools import partialmethod
+import contextlib
+import warnings
+import requests
+
+
+@contextlib.contextmanager
```

Comment:
Did you mean to keep this around? I think it would be preferable just to remove the `filterwarnings` from `pytest.ini` when we create these cassettes

Reply:
There is still the exception raised in Praw, so even after removing the `filterwarnings` I needed to use this to make the requests for the cassette. I figured I might as well check it in so others can use it to record cassettes in the future.